website/content/docs/v0.9/Reference/configuration.md

```diff
@@ -1071,7 +1071,7 @@ Examples:
 
 ``` yaml
 coreDNS:
-    image: k8s.gcr.io/coredns:1.7.0 # The `image` field is an override to the default coredns image.
+    image: k8s.gcr.io/coredns:1.8.1 # The `image` field is an override to the default coredns image.
```

Comment:
looks like official image is on the Docker Hub: https://hub.docker.com/layers/coredns/coredns/1.8.0/images/sha256-10ecc12177735e5a6fd6fa0127202776128d860ed7ab0341780ddaeb1f6dfe61?context=explore